src/main/java/org/casbin/adapter/DynamoDBAdapter.java

```
@@ -0,0 +1,256 @@
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.ItemCollection;
+import com.amazonaws.services.dynamodbv2.document.ScanOutcome;
+import com.amazonaws.services.dynamodbv2.document.Table;
+import com.amazonaws.services.dynamodbv2.document.spec.ScanSpec;
+import com.amazonaws.services.dynamodbv2.model.AttributeDefinition;
+import com.amazonaws.services.dynamodbv2.model.KeySchemaElement;
+import com.amazonaws.services.dynamodbv2.model.KeyType;
+import com.amazonaws.services.dynamodbv2.model.ProvisionedThroughput;
+import com.amazonaws.services.dynamodbv2.model.ScalarAttributeType;
+
+class CasbinRule {
+    String ptype;
+    String v0;
+    String v1;
+    String v2;
+    String v3;
+    String v4;
+    String v5;
+}
+
+/**
+ * DynamoDBAdapter is the AWS DynamoDB adapter for jCasbin 
+ */
+public class DynamoDBAdapter implements Adapter
+{
+    private AmazonDynamoDB client;
+    private DynamoDB dynamoDB;
+    private Table table;
+
+    public static void main(String[] args) {
+        Enforcer e = new Enforcer("examples/rbac_model.conf", "examples/rbac_policy.csv");
+        DynamoDBAdapter a = new DynamoDBAdapter("http://localhost:8000", "cn-north-1");
+        a.savePolicy(e.getModel());
+        a.loadPolicy(e.getModel());
+    }
+
+    public DynamoDBAdapter(String serviceEndpoint, String signingRegion) {
+        try {
+            this.client = AmazonDynamoDBClientBuilder
+                            .standard()
+                            .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(serviceEndpoint, signingRegion))
+                            .build();
+    
+            this.dynamoDB = new DynamoDB(this.client);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void createTable() {
+        try {
```

Remove `try...catch`. We don't should ignore the error.